summingbird-storm/src/test/scala/com/twitter/summingbird/storm/TestAggregateOutpoutCollector.scala

```diff
@@ -0,0 +1,49 @@
+package com.twitter.summingbird.storm
+
+import backtype.storm.spout.{ ISpoutOutputCollector, SpoutOutputCollector }
+import scala.collection.mutable.{ Set => MSet }
+import java.util
+import org.scalacheck._
+
+/**
+ * Created by pnaramsetti on 8/19/16.
+ */
+class TestAggregateOutpoutCollector(in: ISpoutOutputCollector, expected: MSet[(Int, Map[_, _])]) extends SpoutOutputCollector(in) {
```

Comment:
Can we name "expected" more descriptively? We can put whatever this MSet represents into the name, it's not obvious from the types that it represents.